CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking changes
 - Apel plugin: change config parameter `time_db_path` to `time_json_path`
+- Apel plugin: change config format from INI to YAML
```

Comment:
You might want to add your author information (`([@dirksammel](https://github.com/dirksammel))`) to this line (to match our general changelog format). This is also missing in the line above.